dist/Components/relatedListings/index.jsx

```diff
@@ -0,0 +1,24 @@
+/* eslint-disable import/extensions */
+/* eslint-disable react/destructuring-assignment */
+import React from 'react';
+import Related from './related.jsx';
+import NearbyCities from './nearbyCities.jsx';
+
+class RelatedListings extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
```

Comment:
Does this need to be a stateful component?

Reply:
See above